### DISCUSSION TITLE
Removing the gap between title and content

### DISCUSSION THREAD

**Discussion by dotnwat** [jpanther/congo#495] [Q&A]
I'm trying to remove the large gap between the header and the content in the page:

![image](https://user-images.githubusercontent.com/242417/222927981-6228c666-dbbb-4b4f-970e-e57917dd5b9d.png)

In contrast, the Congo docs don't seem to have the same gap:

![image](https://user-images.githubusercontent.com/242417/222927991-308240f7-1090-4c7c-a300-5e3ecffe1f13.png)

My congo installation is effectively out-of-the-box. Any suggestions? Thanks!

**Comment by bob3x255** [comment]
Try taking a look at `themes/congo/layouts/single.html` - the template that is used for your particular page. https://github.com/jpanther/congo/blob/e55ee1ef3bb55e82294e916c5a083daa6bb44242/layouts/_default/single.html#L10
There might be margins that you may want to modify to get the desired outcome. https://tailwindcss.com/docs/margin
Remember, that if you want to change the template itself, it is better to copy the layout you want to modify into your root `layouts` folder. Reading for reference: https://gohugo.io/templates/lookup-order/
